tests/test_binary_quadratic_model.py

```diff
@@ -1311,6 +1311,159 @@ def test_binary_property_relabel(self):
         self.assertIn('a', model.spin.linear)
         self.assertNotIn(0, model.spin.linear)
 
+    def test_dtype_basic_construction_float(self):
+        linear = {'a': 1, 'b': 1., 'c': np.int8(1), 'd': np.float(1)}
+        quadratic = {'ab': 1, 'bc': 1., 'cd': np.uint(16), 'da': np.float64(1)}
```

Comment:
In `linear` and `quadratic`, why are `b` and `bc` marked as "1.", while `a` and `ab` are "1"?